xgcm/grid.py

```diff
@@ -1031,6 +1031,12 @@ def interp(self, da, axis, **kwargs):
         da_i : xarray.DataArray
             The interpolated data
         """
+        if isinstance(axis, list):
+            out = da
+            for axx in axis:
+                out = self.interp(out, axx, **kwargs)
+                #!!!the kwargs needs to be provided in a dict here?
```

Comment:
I am not sure if this is the most elegant way, but it fundamentally works. I am not sure how we would parse the kwargs here. We shurely would want to be able to pass e.g. different boundaries for each axis.